internal/server/storage/postgres/migrations/20231027223251_create_users_table.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE users (
+   id bigserial NOT NULL PRIMARY KEY,
+   email text NOT NULL UNIQUE,
+   password text NOT NULL,
+   created_at timestamp NOT NULL,
```

Comment:
```suggestion
   created_at timestamp NOT NULL DEFAULT now(),
```
и можно вообще выбросить это поле из всех инсертов.